src/index.html

```diff
@@ -11,8 +11,55 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Link to a stylesheet with `.scss` extension won't work directly in HTML. You should compile SCSS to CSS and link to the compiled `.css` file instead.